### PR TITLE
converted html table to vuetable 2

### PR DESCRIPTION
**Pull request by ryanmchristensen**

### CODE REVIEW

**Comment by ericpoe** [comment]
Looks great! It looks like the TravisCI build failed due to a file missing a blank line at the end (PSR-2 https://www.php-fig.org/psr/psr-2/#22-files) something that you can quickly fix with `composer lint-fix` (alias for `vendor/bin/phpcbf`)

**Comment by ryanmchristensen** [comment]
Thanks for the tip!

File: assets/js/views/client/ClientList.vue [hunk 1]
```diff
@@ -68,14 +73,37 @@
         props:[],
         data() {
             return {
-                clients: []
+                clients: [],
+                columns: [
+                    { name: '__slot:link', title: "Client Id", sortField: 'id' },
+                    //todo: find a better way to sort value objects #30
```
Comment (ericpoe) [inline assets/js/views/client/ClientList.vue]: Thank you!

[non-blocking] `// todo: find a better way to sort the value objects @link https://github.com/happybottoms/coverd/issues/30` would turn that into a link that many IDEs and text editors (ok, VSCode) would understand :)

**Comment by ryanmchristensen** [comment]
No problem! Thanks for the opportunity, I look forward to helping in the future :)

**Comment by ericpoe** [comment]
Thank you for the pull request! Cheers

**Pull request merged**